src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -75,18 +75,6 @@
   [accounts address]
   (some #(when (= (:address %) address) %) accounts))
 
-(defn calculate-raw-balance
-  [raw-balance decimals]
-  (if-let [n (utils.number/parse-int raw-balance nil)]
-    (/ n (Math/pow 10 (utils.number/parse-int decimals)))
-    0))
-
-(defn token-value-in-chain
-  [{:keys [balances-per-chain decimals]} chain-id]
-  (let [balance-in-chain (get balances-per-chain chain-id)]
-    (when balance-in-chain
-      (calculate-raw-balance (:raw-balance balance-in-chain) decimals))))
-
```

Comment:
Removed these functions because they were used by none. Also they have an implementation in our money namespace, so better to use that implementation.

Reply:
Nice, your will get merged before, so this change won't take effect